### ISSUE TITLE
Update nagging while the package isn't updated in Windows Store

### ISSUE THREAD

**Issue by NatoBoram** [PowerShell/PowerShell#14776]
PowerShell Preview should not show an update nag if it's impossible to update it.

## Steps to reproduce

1. Install PowerShell Preview from the Windows Store : https://www.microsoft.com/store/productId/9P95ZZKTNRN4
2. Open PowerShell Preview

## Expected behavior

```none
PowerShell 7.2.0-preview.2
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

PS C:\Users\natob>
```

## Actual behavior

```none
PowerShell 7.2.0-preview.2
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

   A new PowerShell preview release is available: v7.2.0-preview.3
   Upgrade now, or check out the release page at:
     https://aka.ms/PowerShell-Release?tag=v7.2.0-preview.3

PS C:\Users\natob>
```

## Environment data

```none
Name                           Value
----                           -----
PSVersion                      7.2.0-preview.2
PSEdition                      Core
GitCommitId                    7.2.0-preview.2
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

---

This issue could be fixed by #14780.

Fixing this issue would resolve these : #13615, #14615, #14772, #14797, #14800

Temporary fix : https://github.com/PowerShell/PowerShell/issues/12701#issuecomment-629702988

**Comment by martingbrown** [comment]
This applies to the non-preview version as well. I'm currently seeing this but can not get Windows Store to update:

> PowerShell 7.1.1
> Copyright (c) Microsoft Corporation.
> 
> https://aka.ms/powershell
> Type 'help' to get help.
> 
>    A new PowerShell stable release is available: v7.1.2
>    Upgrade now, or check out the release page at:
>      https://aka.ms/PowerShell-Release?tag=v7.1.2

It would create a better experience if this nag was either delayed a until the Windows Store is updated or at least changed to give some indication that the update may be delayed.

**Comment by daxian-dbw** [comment]
Close this as a duplicate of #14772
